src/slurm_plugin/instance_manager.py

```diff
@@ -660,36 +657,24 @@ def _add_instances_for_resume_file(
         self._clear_unused_launched_instances()
 
         self._scaling_for_jobs_single_node(
-            job_list=slurm_resume_data.jobs_single_node_no_oversubscribe,
+            job_list=slurm_resume_data.jobs_single_node_no_oversubscribe
+            + slurm_resume_data.jobs_single_node_oversubscribe,
             launch_batch_size=launch_batch_size,
             assign_node_batch_size=assign_node_batch_size,
             update_node_address=update_node_address,
             all_or_nothing_batch=all_or_nothing_batch,
         )
 
         self._scaling_for_jobs_multi_node(
-            job_list=slurm_resume_data.jobs_multi_node_no_oversubscribe,
-            node_list=slurm_resume_data.multi_node_no_oversubscribe,
+            job_list=slurm_resume_data.jobs_multi_node_no_oversubscribe
+            + slurm_resume_data.jobs_multi_node_oversubscribe,
```

Comment:
Same here, having a `jobs_multi_node` property in SlurmResumeData.

Reply:
yes, see other comment